docs/cli-reference.adoc

```diff
@@ -398,6 +406,28 @@ By default, builder images will be used from the current namespace. You can expl
 
 A full list of component types that can be deployed is available using: 'odo catalog list'
 
+[[debug]]
+debug
```

Comment:
Why is debug section empty?

Reply:
TBH, I'm not sure if that's something I should have added or `make generate-cli-reference` should have.

Reply:
OK, no examples in the command-line. It will have to be fixed there.

/lgtm